lib/metadataTypes/DataExtension.js

```diff
@@ -252,13 +252,19 @@ class DataExtension extends MetadataType {
                 const existingFields = DataExtension.oldFields[item[this.definition.nameField]];
                 if (item.Fields !== '' && existingFields) {
                     // TODO should be replaced by a manual sort using existingFields
-                    // ! this is inefficient because it triggers a new download of the fields during the saveResults() step
+                    // ! this is inefficient because it triggers a new download of the fields
```

Comment:
if we leave comments like this, can we consider adding either a task to fix later?

Reply:
Fair point. Creating a task now